From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Arjan van de Ven <arjan@linux.intel.com>
Date: Sat, 6 Aug 2016 21:02:00 +0000
Subject: [PATCH] Set default autocorrect timeout to 1.5 seconds

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 help.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/help.c b/help.c
index 1de9c0d..1181a1c 100644
--- a/help.c
+++ b/help.c
@@ -467,7 +467,7 @@ int is_in_cmdlist(struct cmdnames *c, const char *s)
 	return 0;
 }
 
-static int autocorrect;
+static int autocorrect = 15;
 static struct cmdnames aliases;
 
 static int git_unknown_cmd_config(const char *var, const char *value, void *cb)
